engine/app/components/citizens_advice_components/input.rb

```diff
@@ -32,6 +36,29 @@ def allowed_width_values
       %w[2ch 4ch 8ch 16ch] << nil
     end
 
+    def allowed_type_values
+      %i[
```

Comment:
I wonder if should limit this to text based values? Not sure about this component handling stuff like colour / file types etc. that feel like they should be their own input types in future.